BackendIntegrationTests/BackendIntegrationTests.swift

```diff
@@ -287,18 +287,17 @@ private extension BackendIntegrationTests {
 
     func purchaseMonthlyOffering(completion: ((CustomerInfo?, Error?) -> Void)? = nil) {
         Purchases.shared.getOfferings { offerings, error in
-            expect(error).to(beNil())
+            guard error == nil else { XCTFail("Error fetching offerings: \(error!)"); return }
 
             let offering = offerings?.current
             expect(offering).toNot(beNil())
             
-            let monthlyPackage = offering?.monthly
-            expect(monthlyPackage).toNot(beNil())
+            guard let monthlyPackage = offering?.monthly else { XCTFail("Package not found"); return }
```

Comment:
should this bit be an `XCTUnwrap`?

Reply:
Never mind we can't 😢 
```
@objc func getOfferings(completion: @escaping (Offerings?, Error?) throws -> Void) rethrows
```
> Method cannot be marked `@objc` because the type of the parameter cannot be represented in Objective-C